docs/appendices/release-notes/5.3.6.rst

```diff
@@ -1,17 +1,10 @@
 .. _version_5.3.6:
 
-==========================
-Version 5.3.6 - Unreleased
-==========================
+=============
+Version 5.3.6
+=============
 
-.. comment 1. Remove the " - Unreleased" from the header above and adjust the ==
-.. comment 2. Remove the NOTE below and replace with: "Released on 20XX-XX-XX."
-.. comment    (without a NOTE entry, simply starting from col 1 of the line)
-
-.. NOTE::
-
-    In development. 5.3.6 isn't released yet. These are the release notes for
-    the upcoming release.
+Released on 2023-08-11.
 
 .. NOTE::
```

Comment:
Probably missing smth but where are fixes entries?
I think we need to have a single entry - https://github.com/crate/crate/commit/2ad0da463a32a65ec2b43763979da51a5860813b?

Reply:
There is only one fix: https://github.com/crate/crate/pull/14576/files#diff-0c96cf8ad0ab961e85b4fed6e3ae93c3442514689ba44ed7ada228f1fff0f44fR42

Reply:
sorry, was expecting to see release entries, forgot that ew have them already.